### ISSUE TITLE
e2e test failed

### ISSUE THREAD

**Issue by cwen0** [chaos-mesh/chaos-mesh#1317]
## Bug Report

**What version of Kubernetes are you using?**
  

**What did you do?**

Recently, when to exec `run-e2e-tests`, e2e tests often broken and reported the following error info: 

```
[2020-12-22T06:15:15.084Z] [1] • Failure [67.416 seconds]
[2020-12-22T06:15:15.084Z] [1] [Basic]
[2020-12-22T06:15:15.084Z] [1] /home/jenkins/agent/workspace/chaos-mesh-e2e/go/src/github.com/chaos-mesh/chaos-mesh/test/e2e/chaos/basic.go:50
[2020-12-22T06:15:15.084Z] [1]   [NetworkChaos]
[2020-12-22T06:15:15.084Z] [1]   /home/jenkins/agent/workspace/chaos-mesh-e2e/go/src/github.com/chaos-mesh/chaos-mesh/test/e2e/chaos/basic.go:251
[2020-12-22T06:15:15.084Z] [1]     [NetworkPartition]
[2020-12-22T06:15:15.084Z] [1]     /home/jenkins/agent/workspace/chaos-mesh-e2e/go/src/github.com/chaos-mesh/chaos-mesh/test/e2e/chaos/basic.go:290
[2020-12-22T06:15:15.084Z] [1]       [Schedule] [It]
[2020-12-22T06:15:15.084Z] [1]       /home/jenkins/agent/workspace/chaos-mesh-e2e/go/src/github.com/chaos-mesh/chaos-mesh/test/e2e/chaos/basic.go:291
[2020-12-22T06:15:15.084Z] [1] 
[2020-12-22T06:15:15.084Z] [1]       Dec 22 06:15:13.440: send request from network-peer-0-cc8b8f486-gbr5f to network-peer-1-7bff797cf8-gx78g to test delay failed
[2020-12-22T06:15:15.084Z] [1]       Unexpected error:
[2020-12-22T06:15:15.084Z] [1]           <*url.Error | 0xc000815ef0>: {
[2020-12-22T06:15:15.084Z] [1]               Op: "Post",
[2020-12-22T06:15:15.084Z] [1]               URL: "http://localhost:33102/network/ping",
[2020-12-22T06:15:15.084Z] [1]               Err: {
[2020-12-22T06:15:15.084Z] [1]                   err: "net/http: request canceled (Client.Timeout exceeded while awaiting headers)",
[2020-12-22T06:15:15.084Z] [1]                   timeout: true,
[2020-12-22T06:15:15.084Z] [1]               },
[2020-12-22T06:15:15.084Z] [1]           }
[2020-12-22T06:15:15.084Z] [1]           Post http://localhost:33102/network/ping: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
[2020-12-22T06:15:15.084Z] [1]       occurred
[2020-12-22T06:15:15.084Z] [1] 
[2020-12-22T06:15:15.084Z] [1]       /home/jenkins/agent/workspace/chaos-mesh-e2e/go/src/github.com/chaos-mesh/chaos-mesh/test/e2e/chaos/networkchaos/misc.go:233 

...
[2020-12-22T06:15:13.691Z] [3] [Basic] [DNSChaos] 
[2020-12-22T06:15:13.691Z] [3]   [ERROR]
[2020-12-22T06:15:13.691Z] [3]   /home/jenkins/agent/workspace/chaos-mesh-e2e/go/src/github.com/chaos-mesh/chaos-mesh/test/e2e/chaos/basic.go:335
[2020-12-22T06:15:13.691Z] [3] [BeforeEach] [Basic]
[2020-12-22T06:15:13.691Z] [3]   /go/pkg/mod/k8s.io/kubernetes@v1.17.2/test/e2e/framework/framework.go:151
[2020-12-22T06:15:13.691Z] [3] STEP: Creating a kubernetes client
[2020-12-22T06:15:13.691Z] [3] Dec 22 06:15:13.365: INFO: >>> kubeConfig: /etc/kubernetes/admin.conf
[2020-12-22T06:15:13.691Z] [3] STEP: Building a namespace api object, basename chaos-mesh
[2020-12-22T06:15:13.691Z] [3] STEP: Waiting for a default service account to be provisioned in namespace
[2020-12-22T06:15:13.691Z] [3] [BeforeEach] [Basic]
[2020-12-22T06:15:13.691Z] [3]   /home/jenkins/agent/workspace/chaos-mesh-e2e/go/src/github.com/chaos-mesh/chaos-mesh/test/e2e/chaos/basic.go:62
[2020-12-22T06:15:13.691Z] [3] Dec 22 06:15:13.386: INFO: >>> kubeConfig: /etc/kubernetes/admin.conf
[2020-12-22T06:15:13.691Z] [1] Dec 22 06:15:13.440: FAIL: send request from network-peer-0-cc8b8f486-gbr5f to network-peer-1-7bff797cf8-gx78g to test delay failed
[2020-12-22T06:15:13.691Z] [1] Unexpected error:
[2020-12-22T06:15:13.691Z] [1]     <*url.Error | 0xc000815ef0>: {
[2020-12-22T06:15:13.691Z] [1]         Op: "Post",
[2020-12-22T06:15:13.691Z] [1]         URL: "http://localhost:33102/network/ping",
[2020-12-22T06:15:13.691Z] [1]         Err: {
[2020-12-22T06:15:13.691Z] [1]             err: "net/http: request canceled (Client.Timeout exceeded while awaiting headers)",
[2020-12-22T06:15:13.691Z] [1]             timeout: true,
[2020-12-22T06:15:13.691Z] [1]         },
[2020-12-22T06:15:13.691Z] [1]     }
[2020-12-22T06:15:13.691Z] [1]     Post http://localhost:33102/network/ping: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
[2020-12-22T06:15:13.691Z] [1] occurred
```  

More log:  https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/chaos-mesh-e2e/detail/chaos-mesh-e2e/1027/pipeline

**What did you expect to see?**

**What did you see instead?**

**Output of chaosctl**

**Comment by YangKeao** [comment]
Maybe related with #1279 ?

**Comment by WangXiangUSTC** [comment]
this issue doesn't happen recently, maybe it is fixed in #1393

I close this issue now, and can open it again when meeting the error again
